calyx-py/test/builder_example.py

```diff
@@ -0,0 +1,36 @@
+from calyx.builder import Builder
+
+
+def build():
+    prog = Builder()
+    main = prog.component("main")
+
+    lhs = main.reg("lhs", 32)
+    rhs = main.reg("rhs", 32)
+    sum = main.reg("sum", 32)
+    add = main.add("add", 32)
+
+    with main.group("update_operands") as update_operands:
+        lhs.in_ = 1
+        rhs.in_ = 41
+        lhs.write_en = 1
+        rhs.write_en = 1
+        update_operands.done[lhs.done & rhs.done] = 1
```

Comment:
Is this meant to be `update_operands[done] = lhs.done & rhs.done ? 1'd1`?

Reply:
Yes, that is indeed what it generates. This was my first attempt at finding some kind of Python syntax to abuse for conditional assignments, and you're right that it looks super weird because the square brackets are in the wrong place. I think I have a better solution now!